nominal/__init__.py

```diff
@@ -17,6 +17,7 @@
 )
 from nominal.nominal import (
     checklist_builder,
+    create_asset,
```

Comment:
I havent touched the python client before, but out of curiousity is this import for the test? Since it's not being used by the new endpoints

Reply:
Yeah, we weren't exposing this through the top-level package beforehand. It's not strictly required for the test, we could have just imported `nominal.nominal.create_asset` there, but from an ergonomics standpoint it seems reasonable to expose asset creation in this manner if we expect users to start adding assets to runs more regularly.